frontends/multi/src/router/settings/mod.rs

```diff
@@ -54,8 +54,14 @@ impl Settings {
         font_families.push(String::from("Cascadia Mono (built-in)"));
 
         Settings {
-            default_file_path: rio_backend::config::config_file_path(),
-            default_dir_path: rio_backend::config::config_dir_path(),
+            default_file_path: rio_backend::config::config_file_path()
```

Comment:
one thing that's good since you revisited it, not every path is utf-8. The best might be (haven't tested yet)

```rust
default_file_path: rio_backend::config::config_file_path().into_os_string().into_string().unwrap_or_default()
```

Reply:
(PathBuf might not be utf-8 on Windows for example)

Reply:
much better